projects/ngrx.io/content/guide/router-store/selectors.md

```diff
@@ -13,13 +14,8 @@ Usage:
 
 [Full App Used In This Example](https://stackblitz.com/edit/ngrx-router-store-selectors?file=src/app/car.state.ts)
 
-`router.selectors.ts`
```

Comment:
The changes to the code snippets, migraties the markdown code blocks to `code-example` blocks.